commcare_connect/opportunity/views.py

```diff
@@ -453,3 +457,38 @@ def user_visits_list(request, org_slug=None, opp_id=None, pk=None):
         "opportunity/user_visits_list.html",
         context=dict(opportunity=opportunity, table=user_visits_table, user_name=opportunity_access.display_name),
     )
+
+
+@org_admin_required
+def send_message_mobile_users(request, org_slug=None, pk=None):
+    opportunity = get_object_or_404(Opportunity, pk=pk, organization=request.org)
+    user_ids = (
+        OpportunityAccess.objects.filter(opportunity=opportunity, accepted=True)
+        .prefetch_related("user")
```

Comment:
nit: i dont think you need to prefetch here since you only want the id. I would assume if you put "user_id" in the `values_list`, it will do the right thing without a join at all